### ISSUE TITLE
upgrading 1.0.2 -> 1.1.0 throws parse errors

### ISSUE THREAD

**Issue by mreinstein** [browserify/brfs#23]
when I rely on `brfs@1.0.2` my bundle builds correctly. As soon as I upgrade to `1.1.0` or higher, I start seeing these errors generated:

``` bash
λ npm run watchify

> web@0.0.0 watchify /Users/mike/web
> watchify js/wrapp.coffee -v --extension .js --extension .coffee -o  js/wrapp.js

Error: Parsing file /Users/mike/web/js/models/audit-log.coffee: Line 2: Unexpected identifier
Error: Parsing file /Users/mike/web/js/models/banner.coffee: Line 2: Unexpected identifier
Error: Parsing file /Users/mike/web/js/models/user.coffee: Line 3: Unexpected identifier
```

here's the relevant fields in my package.json:

``` json
"dependencies": {
    "brfs": "1.1.0",
    "browserify": "^3.44.1",
    "browserify-shim": "^3.4.1",
    "coffeeify": "^0.6.0",
    "watchify": "0.8.1"
  },
"scripts": {
    "watchify": "watchify js/wrapp.coffee -v --extension .js --extension .coffee -o assets/javascripts/wrapp.js",
    "browserify": "browserify js/wrapp.coffee --extension .js --extension .coffee -o assets/javascripts/wrapp.js",
  },
"browserify": {
    "transform": [ "coffeeify", "brfs" ]
  }
```

Any ideas why these errors might occur?

**Comment by bernharduw** [comment]
The problem still exists in brfs 1.2.0 in combination with pdfkit. On downgrading to 1.0.2, fonts are loaded, but with 1.2.0 browserify throws.

**Comment by martinheidegger** [comment]
Having that problem still with latest version. Trying to simply load a regular html file in a very short script. Should be reopened.

**Comment by michto01** [comment]
As @martinheidegger and @bernharduw mentioned ... the problem is still there, downgrade works but not the current version

**Comment by dahjelle** [comment]
@JamesKyburz I tried your script, but couldn't reproduce the issue that way. I was still able to reproduce the issue trying to `browserify` `pdfkit`, though.

I *think* it is an issue in `static-module`, as I mentioned in substack/static-module#14, but I'm not sure what the best fix is.

A couple things I noticed when working with `pdfkit` (and please see the `static-module` comment, too):

- Running the `brfs` command-line directly doesn't exhibit the issue.
- If you remove the configuration for `brfs` from in `pdfkit`'s `package.json`, the problem goes away.
- With the `brfs` configuration in `pdfkit`, `brfs` actually runs **twice** on `pdfkit`. The first time, one of the font files is incompletely substituted, leaving an invalid JS file for `brfs` to parse the second time.

While I could get my `pdfkit` test to pass by adding in a `setTimeout` to `static-module` (see the comment there), I'm not at all convinced that's the correct solution.

**Comment by drom** [comment]
Any progress on the issue?

**Comment by goto-bus-stop** [comment]
Fixed in static-module@2.2.5
